server.js

```diff
@@ -1,26 +1,304 @@
+    phone,
+    productivety,
+    site_id,
+    job_id,
+    accesstier,
+    project_id,
+    scrum_id
+  );
+});
+app.post("/deleteEmployee", async (req, res) => {
+  const { id } = req.body;
+  Employee.deleteEmployee(id);
+});
+app.get("/getEmployees", async (req, res) => {
+  let EmployeeTable = Employee.getEmployees();
+  res.send(JSON.stringify({ Employee: EmployeeTable }));
+});
+//************************************** */
+app.post("/createIteration", async (req, res) => {
+  const { name, project_id, index, start_date, end_date } = req.body;
+  Iteration.createIteration(name, project_id, index, start_date, end_date);
+});
+app.post("/editIteration", async (req, res) => {
+  const { id, name, project_id, index, start_date, end_date } = req.body;
+  Iteration.editIteration(id, name, project_id, index, start_date, end_date);
+});
+app.post("/deleteIteration", async (req, res) => {
+  const { id } = req.body;
+  Iteration.deleteIteration(id);
+});
+app.get("/getIterations", async (req, res) => {
+  let IterationTable = Iteration.getIterations();
+  res.send(JSON.stringify({ Iteration: IterationTable }));
+});
+//************************************** */
+app.post("/createJob", async (req, res) => {
+  const { job_id, name } = req.body;
+  Job.createJob(job_id, name);
+});
+app.post("/editJob", async (req, res) => {
+  const { job_id, name } = req.body;
+  Job.editJob(job_id, name);
+});
+app.post("/deleteJob", async (req, res) => {
+  const { job_id } = req.body;
+  Job.deleteJob(job_id);
+});
+app.get("/getJobs", async (req, res) => {
+  let JobTable = Job.getJobs();
+  res.send(JSON.stringify({ Job: JobTable }));
+});
+//************************************** */
+app.post("/createMilestone", async (req, res) => {
+  const { name, date, project_id, description } = req.body;
+  Milestone.createMilestone(name, date, project_id, description);
+});
+app.post("/editMilestone", async (req, res) => {
+  const { id, name, date, project_id, description } = req.body;
+  Milestone.editMilestone(id, name, date, project_id, description);
+});
+app.post("/deleteMilestone", async (req, res) => {
+  const { id } = req.body;
+  Milestone.deleteMilestone(id);
+});
+app.get("/getMilestones", async (req, res) => {
+  let MilestoneTable = Milestone.getMilestones();
+  res.send(JSON.stringify({ Milestone: MilestoneTable }));
+});
+//************************************** */
+app.post("/createProject", async (req, res) => {
+  const { name, start_date, pi_count } = req.body;
+  Project.createProject(name, start_date, pi_count);
+});
+app.post("/editProject", async (req, res) => {
+  const { id, name, start_date, pi_count } = req.body;
+  Project.editProject(id, name, start_date, pi_count);
+});
+app.post("/deleteProject", async (req, res) => {
+  const { id } = req.body;
+  Project.deleteProject(id);
+});
+app.get("/getProjects", async (req, res) => {
+  let ProjectTable = Project.getProjects();
+  res.send(JSON.stringify({ Project: ProjectTable }));
```

Comment:
This could be a good way to test your code locally. Actually better is writing unit test. It could be also an example how to use the models. In any way we should not touch `server.js` in this task.